### PR TITLE
[PT-BR] Add Installation translation

### PR DESCRIPTION
**Pull request by daviirodrig**
- Add pt-br translation for `Installation` article

### CODE REVIEW

File: wiki/Installation/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+## Requisitos mínimos do sistema
+
+- .NET framework 4+ (o instalador deve pedir para fazer o download, se não, pode [encontrá-lo aqui](https://www.microsoft.com/en-us/download/details.aspx?id=48130))
+- Uma placa de vídeo decente.
+
+## Instalando osu!
+
+1. Vá para [página de download](https://osu.ppy.sh/home/download) e baixe o instalador.
+2. Localize e inicie o instalador.
+3. Enquanto o instalador está rodando, existe um tempo de dez segundos que permite você selecionar o local de instalação.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:16]:
```suggestion
3. Enquanto o instalador estiver rodando, há um período de dez segundos que permite você selecionar o local de instalação.
```

File: wiki/Installation/pt-br.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+   - Se quiser mudar, clique no texto embaixo da palavra `Hi!` e selecione o local de instalação.
+   - osu! será instalado nessa localização quando o tempo acabar.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:18]:
```suggestion
   - osu! será instalado no local escolhido quando o tempo acabar.
```

File: wiki/Installation/pt-br.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+4. Quando o instalador completar, osu! irá iniciar. osu!direct será usado para baixar alguns beatmaps de iniciante para você.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:19]:
```suggestion
4. Quando o instalador concluir, o osu! irá iniciar. osu!direct será usado para baixar alguns beatmaps de iniciante para você.
```

File: wiki/Installation/pt-br.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+5. osu! irá pedir para você entrar ou se registrarwill prompt you to sign in or register.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:20]:
```suggestion
5. o osu! irá pedir para você entrar ou se registrar.
```
Little oops.

File: wiki/Installation/pt-br.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+   - Se já tem uma conta, entre.
+   - Se precisa de se registrar, veja [Registro](/wiki/Registration).
+
+## Solução de problemas
+
+Se você está usando Windows 7 64-bit, você pode ter dificuldade instalando osu! corretamente. Tente instalar a versão 64-bit do .NET Framework, caso ele não venha com o instalador do osu!. [Baixe aqui](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:26]:
```suggestion
Se você está usando o Windows 7 64-bit, você pode ter dificuldades em instalar o osu! corretamente. Tente instalar a versão 64-bit do .NET Framework, caso ele não venha com o instalador do osu!. [Baixe aqui](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
```

File: wiki/Installation/pt-br.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+---
+
+**Se ocorrer algum erro ao instalar o jogo, você pode tentar algumas coisas:**
+
+- Tente reinstalar a última versão do .NET Framework. [Baixe aqui](https://dotnet.microsoft.com/download).
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:32]:
```suggestion
- Tente reinstalar a última versão do .NET Framework. [Encontre-o aqui](https://dotnet.microsoft.com/download).
```

File: wiki/Installation/pt-br.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+- Limpe seu PC - Escaneie com anti-virus, limpe o registro. Um bom programa para limpeza geral é CCleaner. Escaneie seus arquivos e seu registro. O instalador pode falhar se existe entradas no registro de uma cópia antiga do osu! no seu sistema.
+
+## Adicionando beatmaapsets
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:35]:
```suggestion
## Adicionando beatmapsets
```
Little oops

File: wiki/Installation/pt-br.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+*Veja também: [Beatmaps](/wiki/Beatmaps)*
+
+Existe duas formas de adicionar beatmaps, via website ou osu!direct (requer [osu!supporter](/wiki/osu!supporter)).
+
+### Website
+
+1. Vá para [página de listagem de beatmaps](https://osu.ppy.sh/beatmapsets) e entre com sua conta.
+2. Encontre uma música que você gostaria de jogar.
+3. Clique no ícone de download para salver o beatmapset em seu computador.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:45]:
```suggestion
3. Clique no ícone de download para salvar o beatmapset em seu computador.
```

File: wiki/Installation/pt-br.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+4. Localize e abra o arquivo do beatmapset, ele tem a extensão `.osz`
+   - Alternativamente, você também pode colcoar o arquivo do beatmapset na pasta `Songs` da instalação do osu!
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:47]:
```suggestion
   - Alternativamente, você também pode colocar o arquivo do beatmapset na pasta `Songs` da instalação do osu!
```

File: wiki/Installation/pt-br.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+5. osu! vai cuidar do resto e carregar o beatmapset.
+   - Se, por algum motivo, você não o vê, pressione `F5` na [Seleção de música](/wiki/Interface#song-selection-screen) para reprocessar seus beatmaps.
+
+### osu!direct
+
+*Nota: Você precisa de uma tag osu!supporter e uma conexão com a internet para usar osu!direct.*
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:53]:
```suggestion
*Nota: Você precisa de uma tag osu!supporter e uma conexão com a internet para usar o osu!direct.*
```

File: wiki/Installation/pt-br.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+1. Abra osu! e clique no botão osu!direct no lado direito da tela principal.
+2. Encontre uma música que gostaria de jogar.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:56]:
```suggestion
2. Encontre uma música que você gostaria de jogar.
```
Consistency with translation above

File: wiki/Installation/pt-br.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+3. Dependendo como o skinner compactou sua skin, você precisa fazer coisas diferentes.
+   - Se a skin usa `.osk`:
+     1. Abra ou clique e arraste para dentro do osu!.
+     2. osu! irá processar a skin automaticamente e selecionar ela.
+   - Se a skin está compressada (utiliza `.7z`, `.rar`, `.zip`, etc.):
+     1. Extraia.
+     2. Mova a pasta da skin extraída para o diretório `Skins` da instalação do osu!.
+     3. Se osu! estiver aberto, a skin aparecerá na lista apenas após você pressionar `Ctrl` + `Alt` + `Shift` + `S`.
+     4. No osu!, vá em [Opções](/wiki/Options) e procure por `Skin`.
+     5. Desça até a opção `Current skin:` e selecione a skin que baixou.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:77]:
```suggestion
     5. Desça até a opção `Selecionar skin:` e selecione a skin que baixou.
```
Consistency with in-game PT-BR client translation

File: wiki/Installation/pt-br.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+
+- .NET framework 4+ (o instalador deve pedir para fazer o download, se não, pode [encontrá-lo aqui](https://www.microsoft.com/en-us/download/details.aspx?id=48130))
+- Uma placa de vídeo decente.
+
+## Instalando osu!
+
+1. Vá para [página de download](https://osu.ppy.sh/home/download) e baixe o instalador.
+2. Localize e inicie o instalador.
+3. Enquanto o instalador está rodando, existe um tempo de dez segundos que permite você selecionar o local de instalação.
+   - Se quiser mudar, clique no texto embaixo da palavra `Hi!` e selecione o local de instalação.
```
Comment (Flantsune) [inline wiki/Installation/pt-br.md:17]:
```suggestion
   - Se quiser mudar, clique no texto embaixo da palavra `Olá!` e selecione o local de instalação.
```
The installer actually changes to PT-BR if it detects the locale after downloading it, it is translated there as well. Just checked.

**Comment by daviirodrig** [comment]
Wow that's a lot of typos I shouldn't translated this at 4am haha
Anyways, thanks for the review I'll sum up everything in one commit.

**Pull request merged**